libs/ffi/mitlsffi.h

```diff
@@ -173,9 +192,12 @@ typedef struct {
   // only used by the client
   const char *host_name; // Client only, sent in SNI. Can pass NULL for server
   const quic_ticket *server_ticket; // May be NULL
+  const mitls_extension *exts; // Array of custom extensions to offer, may be NULL
+  size_t exts_len;
```

Comment:
Rename to exts_count? Makes more sense if it is the count of extensions rather than the byte-length of the array of exts.